manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,65 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+You can find the instances in [Pipelines Console](http://pantheon.corp.google.com/ai-platform/pipelines).
```

Comment:
Suggested text: You can find Kubeflow Pipelines instances deployed via Marketplace in [Pipelines Console]